packages/app-headless-cms/src/admin/plugins/fields/richText.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { ReactComponent as NotesIcon } from "./icons/notes-black-24px.svg";
+import { Grid, Cell } from "@webiny/ui/Grid";
+import { I18NInput } from "@webiny/app-i18n/admin/components";
+import { FbBuilderFieldPlugin } from "@webiny/app-headless-cms/types";
+import { i18n } from "@webiny/app/i18n";
+const t = i18n.ns("app-headless-cms/admin/fields");
+
+const plugin: FbBuilderFieldPlugin = {
+    type: "content-model-editor-field-type",
+    name: "content-model-editor-field-type-richText",
+    field: {
+        type: "rich-text",
+        label: t`Rich text`,
+        description: t`Text formatting with references and media`,
+        icon: <NotesIcon />,
+        validators: ["required"],
```

Comment:
Remove this one for now, it won't work because basically the validator checks if a value is falsy, and this component will at least store `[{}]` I believe. We can revisit this one later.

Reply:
Yeah. You're right. Removing it :+1: